src/test/regress/expected/with.out

```diff
@@ -2660,3 +2660,87 @@ SELECT * FROM cte JOIN (SELECT * FROM d JOIN cte USING (a) LIMIT 1) d_join_cte U
 
 DROP TABLE d;
 DROP TABLE r;
+-- Test that planner doesn't use direct dispatch in case of shared CTE.
```

Comment:
The test description contradicts with test output. 

PS. Also, actualize PR description.

Reply:
Done